src/include/wiredtiger.in

```diff
@@ -2664,18 +2664,19 @@ struct __wt_connection {
      * @configstart{WT_CONNECTION.query_timestamp, see dist/api_data.py}
      * @config{get, specify which timestamp to query: \c all_durable returns the largest timestamp
      * such that all timestamps up to and including that value have been committed (possibly bounded
-     * by the application-set \c durable timestamp); \c last_checkpoint returns the timestamp of the
-     * most recent stable checkpoint; \c oldest_timestamp returns the most recent \c
-     * oldest_timestamp set with WT_CONNECTION::set_timestamp; \c oldest_reader returns the minimum
-     * of the read timestamps of all active readers; \c pinned returns the minimum of the \c
+     * by the application-set \c durable timestamp); \c backup_checkpoint returns the stable
+     * timestamp of the checkpoint pinned for an open backup cursor; \c last_checkpoint return the
```

Comment:
Change "last_checkpoint return" to "last_checkpoint returns" like it was before.